crates/hyvectl/src/util.rs

```diff
@@ -1,12 +1,16 @@
+use futures::stream::BoxStream;
 use hyveos_sdk::Connection;
 
-use crate::error::HyveCtlResult;
-use crate::out::CommandOutput;
-use futures::stream::BoxStream;
+use crate::{error::HyveCtlResult, out::CommandOutput};
 
-pub trait CommandFamily {
-    async fn run(self, connection: &Connection)
-        -> BoxStream<'static, HyveCtlResult<CommandOutput>>;
+pub trait CommandFamily: Sized {
+    async fn run(self, _: &Connection) -> BoxStream<'static, HyveCtlResult<CommandOutput>> {
+        unimplemented!()
+    }
+
+    async fn init(self) -> HyveCtlResult<CommandOutput> {
+        unimplemented!()
+    }
```

Comment:
I don't like this. Could you just not implement `CommandFamily` for the `init` command instead?